src/main/scala/tech/cryptonomic/conseil/tezos/michelson/renderer/MichelsonRenderer.scala

```diff
@@ -54,6 +54,6 @@ object MichelsonRenderer {
         .map(_.render())
         .mkString(" ;\n")
         .lines
-        .mkString("\n" + indent)
+        .mkString(s"\n${indent}")
```

Comment:
@dorians I had to change this line because I was getting a compiler error before: 

```
[error] /home/vishakh/dev/cryptonomic/Conseil/src/main/scala/tech/cryptonomic/conseil/tezos/michelson/renderer/MichelsonRenderer.scala:59:10: value mkString is not a member of java.util.stream.Stream[String]
[error] possible cause: maybe a semicolon is missing before `value mkString'?
[error]         .mkString("\n" + indent)
[error]          ^
```

Could you tell me how to fix it peoperly?

Reply:
The issue is with the `.lines` call, which I think we have fixed before in previous PRs.

The jdk9+ introduced a `.lines` method that conflicts with the existing one from scala, and returns a type that's obviously different, so that `mkString` is not available.

tl;dr; you only need to change `.lines` with `linesIterator`